module/apmot/span_test.go

```diff
@@ -0,0 +1,58 @@
+package apmot
```

Comment:
This file is failing the `goimports` check in CI, as the imports aren't grouped. Also, it'll fail the license check, as it's missing a header. You can fix these two things by running `make update-licenses fmt`